%YAML:1.0
#TRACKER PARAMETERS
klt_win_size: 24
klt_max_iter: 30
klt_max_level: 4
klt_eps: 0.1
maxFeatureAge: 25
maxFeaturesPerFrame: 100
quality_level: 0.001
min_distance: 10
block_size: 3
use_harris_detector: false
k: 0.04
equalizeImage: false
nominalBaseline: 0.11
toleranceTemplateMatching: 0.15
templ_cols: 101 #must be odd
templ_rows: 11
stripe_extra_rows: 0
minPointDist: 0.1
maxPointDist: 15
bidirectionalMatching: false
subpixelRefinementStereo: false
featureSelectionCriterion: 0
featureSelectionHorizon: 3
featureSelectionNrCornersToSelect: 100
featureSelectionImuRate: 0.005
featureSelectionDefaultDepth: 5.0
featureSelectionCosineNeighborhood: 0.98 # Rad
featureSelectionUseLazyEvaluation: true
useSuccessProbabilities: 0
useRANSAC: true
minNrMonoInliers: 10
minNrStereoInliers: 5
ransac_threshold_mono: 1e-06
ransac_threshold_stereo: 1
ransac_use_1point_stereo: true
ransac_use_2point_mono: true
ransac_max_iterations: 100
ransac_probability: 0.995
ransac_randomize: false
intra_keyframe_time: 0.2
minNumberFeatures: 100
useStereoTracking: 1
display_time: 100
disparityThreshold: 1
